### DISCUSSION TITLE
How can I test Apps?

### DISCUSSION THREAD

**Discussion by YunusEmreAlps** [uvdesk/community-skeleton#599] [Q&A] [answered]
Hello,

I hope you're well and everything is good for you.

How can I test Apps (Free and Paid) in my Localhost?

Thank you very much.
Best regards

**Comment by PeopleInside** [comment]
Hi, I think you can test only free apps if you have not yet bought a paid app.
For the localhost you need a reply from a UVdesk staff or developer, I don't remember if there is a guide for use UVdesk at localhost.

**Reply by YunusEmreAlps** [reply]
@PeopleInside Thank you for your all helps. I think no chance to test these apps. Because these apps are only used by the SAAS  platforms. (By the way no guide for localhost)

**Reply by PeopleInside** [reply]
If you want info about paid apps https://forums.uvdesk.com/topic/2010/additional-apps-that-you-can-add-to-the-open-source-uvdesk-helpdesk-system you can maybe also start a chat with the support. Maybe a good feedback and idea would be suggest them to have a demo for all this paid apps so user can test online.

**Reply by komal-sh-27** [reply]
@YunusEmreAlps 

Nice day to you!!
We would like to inform you that to make use of the application in the open source version, you have to purchase the plugin from the Webkul store. You can check [here](https://store.webkul.com/UVdesk/UVdesk-Open-Source.html). 
As these plugins are the paid plugin in the open source version of UVdesk.

Also, you can use saas uvdesk for these apps are free. 
You can check or sign up from [here](https://www.uvdesk.com/en/).

Do let us know if having any other kind of concern or query. You can create a ticket from [here](https://support.uvdesk.com/en/customer/create-ticket/).

**Reply by YunusEmreAlps** [reply]
When I compare Zammad and UVDesk. Zammad gives so many features and opportunities for my development process. Wouldn't making some apps open source increase the demand for that app?

I want to add so many features UVDesk and I want to personalize this app. Some of the Apps do not have a version of Open Source for example SLA, Notes on Customer, etc... 

I'm gonna work on adding these features to open source version. Are there any chances that some apps are open source?

**Reply by Aanchal542** [reply, marked as answer]
@YunusEmreAlps 

Hope you doing well!!
When I compare Zammad and UVDesk. Zammad gives so many features and opportunities for my development process. Wouldn't making some apps open source increase the demand for that app?

>> We would like to inform you that we do provide the limited features in the Open source variant, as it is the free version and also has sufficient features and functionality to deliver the best customer support.
Furthermore, we do have multiple applications available but that would be paid one.
But in the Saas variant, you will get these applications for free.

I want to add so many features UVDesk and I want to personalize this app. Some of the Apps do not have a version of Open Source for example SLA, Notes on Customer, etc...

>> If you want any modifications in the current flow of UVdesk, then that would be part of customization (paid service), and the apps which you have mentioned above are comes up as paid extensions in the open source version.

I'm gonna work on adding these features to the open source version. Are there any chances that some apps are open source?

>> As of now, to make use of these applications you have to purchase the plugin from the Webkul store. 

Do let me know if have any other concerns or queries, or you can simply write us an email at support@uvdesk.com accordingly our sales support team will get back to you.
